src/features/collaborator-avatars/CollaboratorList.js

```diff
@@ -51,9 +50,9 @@ class CollaboratorList extends React.Component<Props> {
         const { collaborators, onDoneClick, maxCollaboratorListSize, trackingProps } = this.props;
         const { usernameProps, emailProps, manageLinkProps, viewAdditionalProps, doneButtonProps } = trackingProps;
         const manageAllBtn = (
-            <PlainButton className="manage-all-btn" type="button">
+            <span className="manage-all-btn">
```

Comment:
while this class name is not on a button anymore, it cannot be changed since the classname is used in stylesheets in the main app and the monolith